react/RelatedProducts.js

```diff
@@ -82,7 +82,7 @@ export default class RelatedProducts extends Component {
     }
 
     return (
-      <div className="vtex-related-products">
+      <div className={`${shelf.relatedProducts}`}>
```

Comment:
You don't need to use string interpolation here, you can just do `className={shelf.relatedProducts}`

Reply:
done